web/profiles/custom/os2loop/modules/os2loop_lists/templates/os2loop-lists-expert.html.twig

```diff
@@ -0,0 +1,25 @@
+{#
+/**
+ * @file
+ * Display list of content that the logged in user should be an expert on.
+ *
+ * Available variables:
+ * - data: A list of nodes
+ */
+#}
+{% if data is empty %}
+  <div class="list-group">
+    <div class="list-group-item mb-2 bg-light border-0">
+      {{ 'No content related to your user'|t }}
+    </div>
+  </div>
+{% else %}
+  {% for node in data %}
+    <div class="list-group">
+      <article class="list-group-item mb-2 bg-light border-0">
+        <div class="text-muted small">{{ node.type.entity.label }} </div>
+        <h2 class="h6"><a href="{{ url('entity.node.canonical', { 'node': node.id() }) }}">{{ node.title() }}</a></h2>
```

Comment:
`h2` element with class name `h6`? Is that a typo?

Reply:
Hmm. This should be h3.
H3 is the correct html tag. H6 is a bootstrap helper class to resize the header, instead of styling it.

Reply:
CHanged to h3